From 03d724c3a0f4f8b8d8bbd03eb1d5e961b5bcde82 Mon Sep 17 00:00:00 2001
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Thu, 25 Feb 2016 21:30:06 +0100
Subject: [PATCH] Trim linux.enea.com mirror URL.

FIXME: squash to commit adding these the first time
sed -i 's/repo\/opnfv\/armband\/mos-repos/mos-repos/g'
---
 nailgun/nailgun/fixtures/openstack.yaml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/nailgun/nailgun/fixtures/openstack.yaml b/nailgun/nailgun/fixtures/openstack.yaml
index b252fbf..6ab5779 100644
--- a/nailgun/nailgun/fixtures/openstack.yaml
+++ b/nailgun/nailgun/fixtures/openstack.yaml
@@ -2137,25 +2137,25 @@
                 priority: null
               - type: "deb"
                 name: "mos"
-                uri: "https://linux.enea.com/repo/opnfv/armband/mos-repos/ubuntu/{cluster.release.environment_version}/"
+                uri: "https://linux.enea.com/mos-repos/ubuntu/{cluster.release.environment_version}/"
                 suite: "mos8.0"
                 section: "main restricted"
                 priority: 1050
               - type: "deb"
                 name: "mos-updates"
-                uri: "https://linux.enea.com/repo/opnfv/armband/mos-repos/ubuntu/{cluster.release.environment_version}/"
+                uri: "https://linux.enea.com/mos-repos/ubuntu/{cluster.release.environment_version}/"
                 suite: "mos8.0-updates"
                 section: "main restricted"
                 priority: 1050
               - type: "deb"
                 name: "mos-security"
-                uri: "https://linux.enea.com/repo/opnfv/armband/mos-repos/ubuntu/{cluster.release.environment_version}/"
+                uri: "https://linux.enea.com/mos-repos/ubuntu/{cluster.release.environment_version}/"
                 suite: "mos8.0-security"
                 section: "main restricted"
                 priority: 1050
               - type: "deb"
                 name: "mos-holdback"
-                uri: "https://linux.enea.com/repo/opnfv/armband/mos-repos/ubuntu/{cluster.release.environment_version}/"
+                uri: "https://linux.enea.com/mos-repos/ubuntu/{cluster.release.environment_version}/"
                 suite: "mos8.0-holdback"
                 section: "main restricted"
                 priority: 1100
-- 
1.9.1

